.travis/base.sh

```diff
@@ -12,7 +12,7 @@
 # limitations under the License.
 #
 
-## this is a script soley for inclusion in the main build scripts.
+## this is a script solely for inclusion in the main build scripts.
 ## Common functions that are executed both all types of builds
```

Comment:
I think this line has an explanation error. You should review it from the mentors and fix it as well.

Reply:
i think its self explanatory.